### ISSUE TITLE
why I can't get similar results in the paper

### ISSUE THREAD

**Issue by LiuAndYang** [cheng6076/SNLI-attention#10]
I run this experiments following instructions several times but only get accuracy around 0.800, should I do some changes in the codes. It seems the codes correctly run.

did the following prints means the accracy ?
evaluating loss over split index 3	
test_loss = 0.7915

**Comment by cheng6076** [comment]
I have already given instructions how you can get numbers around 86. If you cannot get anything close, please check your dataset, what processing you have done, what changes you have made. This is not the right place to ask this question.

**Comment by LiuAndYang** [comment]
Ok, thx. I will try that. Sorry for this post.
